apps/api/src/app.module.ts

```diff
@@ -60,9 +61,7 @@ const enterpriseImports = (): Array<Type | DynamicModule | Promise<DynamicModule
     if (require('@novu/ee-billing')?.BillingModule) {
       modules.push(require('@novu/ee-billing')?.BillingModule.forRoot());
     }
-    if (require('./app/support/support.module')?.SupportModule) {
-      modules.push(require('./app/support/support.module')?.SupportModule);
-    }
+    modules.push(SupportModule);
```

Comment:
👏

i think it is safe to move this to baseModules list as it is not part of the enterprise imports

Reply:
Its not part of enterprise private packages but I think its an enterprise import, because if I would put it to the baseModules, then it would be also imported for OS users and they likely don't want our support platform setup.

Reply:
Oh, that's a valid point! i missed the if statement from the outer scope.